filter/exec_filter.go

```diff
@@ -0,0 +1,60 @@
+package filter
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+	"strings"
+	"sync"
+	"text/template"
+
+	"github.com/dgnorton/norobo"
+)
+
+type ExtFilter struct {
+	cmd  string
+	args string
+}
+
+func NewExtFilter(cmd, args string) *ExtFilter {
+	return &ExtFilter{
+		cmd:  cmd,
+		args: args,
+	}
+}
+
+func (e *ExtFilter) Check(c *norobo.Call, result chan *norobo.FilterResult, cancel chan struct{}, done *sync.WaitGroup) {
+	go func() {
+		defer done.Done()
+
+		var cmdArgs bytes.Buffer
+
+		tmpl, err := template.New("args").Parse(e.args)
+		if err != nil {
+			panic(err)
+		}
+
+		err = tmpl.Execute(&cmdArgs, c)
+		if err != nil {
+			panic(err)
+		}
+
+		fmt.Println(cmdArgs.String())
+
+		// Create command
+		out, err := exec.Command(e.cmd, strings.Split(cmdArgs.String(), " ")...).Output()
```

Comment:
Okay for now but would be better if it started the command using `Start` so that it doesn't wait for the process to finish before returning. Then select on output, timeout, and cancel. On a timeout or cancel, it would kill the process.